### ISSUE TITLE
SA1201 not reported in records

### ISSUE THREAD

**Issue by sloscialo** [DotNetAnalyzers/StyleCopAnalyzers#3524]
Version 1.1.118

Ordering rules don't seem to apply for records.

```
public class SimpleClass
{
    public string Name { get; set; }

    private int number;
}

public record SimpleRecord
{
    public string Name { get; set; }

    private int number;
}
```

Expected that SA1201 would be reported for `SimpleRecord` as it is for `SimpleClass`.

**Comment by sloscialo** [comment]
Sorry, this looks fixed when I grabbed the latest beta (1.2.0-beta.435).
